### DISCUSSION TITLE
OpenMM and PBCs

### DISCUSSION THREAD

**Discussion by sef43** [ACEsuit/mace#90] [General]
Hi just highlighting this: https://github.com/felixmusil/torch_nl/issues/1#issuecomment-1467823500

OpenMM does not pass wrapped coordinates to the torchforce to you will need to be careful using torch_nl for the PBCs!

I have a version here that I think works with PBCs:
https://github.com/sef43/openmm-ml/tree/mace/examples/mace

**Comment by jharrymoore** [comment]
Hi Stephen, many thanks for this.  We have moved away from the torch_nl neighbour list for exactly this reason, currently I have your NL implementation in my fork of openmm-ml and the default, which seems to preform pretty well, even compared to the NNPops implementation

**Comment by sef43** [comment]
great! also here is a relevant comment from over in the nequip repo: https://github.com/mir-group/nequip/issues/288#issuecomment-1475964360.
only with more than ~1000 atoms  does the NNPOps implementation becomes faster than naive torchscript
And here is a notebook with the implementations: https://colab.research.google.com/drive/1Vz4QN8Y_cEUkCL2YxY86QhSruYPxfwtr?usp=sharing
